CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
 # Changelog
 
+## [1.1.0](https://github.com/parallaxsecond/parsec/tree/1.1.0) (2022-09-13)
 ## [1.1.0-rc2](https://github.com/parallaxsecond/parsec/tree/1.1.0-rc2) (2022-09-13)
```

Comment:
Hmm, maybe you can make the RC nested headings within the full release? As in, `###` instead of `##`

Something that came to mind just now...

Reply:
Though I'm afraid that the tool might rewrite them next time 😭

Reply:
The tool will rewrite it next time but that's fine because the tag will be moved after this merge. Do you see an issue with that?
The only way I see to have this merge in the change log in this release is 
- Merging this, whatever the format is 
- Moving the tag
- Changing the change log again and force push

Reply:
Why do you need the last change to the Changelog?

Reply:
In order to have this MR included in the Changelog if we want
If it's fine then ... we can edit manually this change log but then in the next release it will be modified

Reply:
Your suggestion looks good 👍🏼 ignore the above comments 😸